dashboard/src/views/Projects/Versions/index.tsx

```diff
@@ -131,15 +130,18 @@ export const VersionsOverview = () => {
             </tr>
           </thead>
 
-          <tbody>
-            {versionsList.versions.map((version) => (
-              <TableRow
-                key={version.id}
-                name={version.name}
-                createdAt={version.createdAt}
-              />
-            ))}
-          </tbody>
+          {versionsList && (
```

Comment:
and if no list is available? or is loading?

Reply:
i'm planning to add placeholders in one of the next tasks. The case with no list is handled in one of the next PRs